tasks/section1.yml

```diff
@@ -1320,7 +1335,7 @@ print $NF }'"
       - rule_1.7.2
 
 - name: "NOTSCORED | 1.8 | AUDIT | Ensure updates, patches, and additional security software are installed"
-  command: /bin/true
+  command: yum check-update
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*